DirectProgramming/DPC++FPGA/Tutorials/Features/prototype/CSR_and_streaming_interfaces/src/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+# To see a Makefile equivalent of this build system:
+# https://github.com/oneapi-src/oneAPI-samples/blob/master/DirectProgramming/DPC++/ProjectTemplates/makefile-fpga
+
+set(SOURCE_FILE CSR_and_streaming_interfaces.cpp)
+set(TARGET_NAME CSR_and_streaming_interfaces)
+set(EMULATOR_TARGET ${TARGET_NAME}.fpga_emu)
+set(SIMULATOR_TARGET ${TARGET_NAME}.fpga_sim)
+
+# FPGA device selection
+if(NOT DEFINED FPGA_BOARD)
+    set(FPGA_BOARD "Arria10")
```

Comment:
If A10 if the default board, then you should update the top of the README to make it so that USM is not a requirement

Reply:
see above discussion about A10